stdlib/3/builtins.pyi

```diff
@@ -760,6 +761,39 @@ class property:
     def fset(self, value: Any) -> None: ...
     def fdel(self) -> None: ...
 
+class code:
```

Comment:
This shouldn't be here, because there is no `code` in `builtins` at runtime. (That's also true for function and module, but they're here for legacy reasons.)

Besides, this duplicates `types.CodeType`.

Reply:
Ugh, you're right. I did look there in python library (not typeshed), but read it as "an object defining a code attribute", since I'm still pretty new to the typing module.